RN Update v13.0.3 & V13.0.4 (#1539)
mia-platform/documentation: docs/release-notes/v13.0.3.md

For more information, visit the related [documentation page](/infrastructure/self-hosted/installation-chart/helm-values/80_whitelabeling.md).

### Compare changes at Deploy on Projects with the enhanced workflow on On-Premise installations

:::caution

> This feature has a malfunction that causes an unexpected error when using *GitLab* and *Jenkins* as CI/CD providers on the project.

Reviewer: ```suggestion
This feature has an unexpected behaviour when using *GitLab* and *Jenkins* as CI/CD providers on the project.

```

> Resolution of these issues will be addressed with future versions

Reviewer: We should add to contact the Mia-Platform referent if you need to disable the feature in the on premise installation

Author: right
